StreamChatUITestsAppUITests/Robots/ParticipantRobot.swift

```diff
@@ -59,10 +59,10 @@ final class ParticipantRobot: Robot {
         return self
     }
     
-    // Think twice before using it
+    // Sleep in milliseconds
     @discardableResult
-    func chill(duration: UInt32) -> Self {
-        sleep(duration)
+    func chill(duration: useconds_t) -> Self {
+        usleep(duration)
```

Comment:
Isn't `wait` the most common pattern here? `chill` can mean many things 
```suggestion
    func wait(duration: useconds_t) -> Self {
        usleep(duration)
```